cmd/podman/system/service_abi.go

```diff
@@ -21,6 +21,11 @@ func restService(opts entities.ServiceOptions, flags *pflag.FlagSet, cfg *entiti
 		err      error
 	)
 
+	if logrus.GetLevel() < logrus.InfoLevel {
+		logrus.Debugf("Logging level upgraded from %q to %q", logrus.GetLevel().String(), logrus.InfoLevel.String())
+		logrus.SetLevel(logrus.InfoLevel)
+	}
+
```

Comment:
I'm not sure where, but this feels like it should be documented somewhere.  I can see some admin trying to reset the level and having it go back up.

Reply:
Added to man page.